PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeRequest.java

```diff
@@ -203,6 +203,21 @@ public void setReturnUrl(@NonNull String returnUrl) {
         this.returnUrl = returnUrl;
     }
 
+    @Nullable
+    public String getUserAuthenticationEmail() {
+        return authEmail;
+    }
+
+    /**
+     * Optional: Sets the email of the user prior to calling authentication
+     *
+     * @param authEmail the users email address
+     */
+    public void setUserAuthenticationEmail(@NonNull String authEmail) {
```

Comment:
```suggestion
    public void setUserAuthenticationEmail(@Nullable String authEmail) {
```
I believe the annotations need to match up for Kotlin interop.

Reply:
Good catch!